Projects/Payment/Sources/Screens/Adyen/AdyenPayIn.swift

```diff
@@ -67,8 +67,10 @@ struct AdyenPayIn: Presentable {
                 }
             }
         } onSuccess: {
-            store.update(query: GraphQL.PayInMethodStatusQuery()) { (data: inout GraphQL.PayInMethodStatusQuery.Data) in
-                data.payinMethodStatus = .active
+            store.withinReadWriteTransaction { transaction in
+                try? transaction.update(query: GraphQL.PayInMethodStatusQuery(), { (data: inout GraphQL.PayInMethodStatusQuery.Data) in
```

Comment:
should we handle errors here?

Reply:
no, there is really nothing to do then, I don't even get why this can throw 🤔